src/nxdoc/nuxeo-server/platform-services/automation/automation-scripting.md

```diff
@@ -228,7 +228,7 @@ Watch the related course on Hyland University:
 
 ## Requirements
 
-Automation Scripting is using Java 8 Nashorn engine. To leverage all the Automation Scripting feature, the Nuxeo server needs to run **JDK 11**.
+Automation Scripting is using Java 8 Nashorn engine. To leverage all the Automation Scripting feature, the Nuxeo server needs to run **JDK 17**.
```

Comment:
I think we can delete this whole section as Nashorn come as a dependency since the upgrade to Java 17, so no strict requirement on a java version.